arte_downloader.sh

```diff
@@ -31,9 +31,9 @@ read -p "$(tput setaf 4)Choose quality (1-$NUMBER_OF_QUALITIES) $(tput sgr0)" CH
 SELECTED_FILE=$(echo $FILES | jq '.['$CHOSEN_QUALITY-1']')
 SELECTED_URL=$(echo $SELECTED_FILE | jq -r '.url')
 SELECTED_FORMAT=$(echo $SELECTED_FILE | jq -r '.mediaType')
-CLEANED_NAME=$(echo $TITLE | sed -e 's/[^A-Za-z0-9._-]/_/g')
+CLEANED_NAME=$(echo $TITLE | sed -e 's/[:\/]/_/g')
```

Comment:
```suggestion
CLEANED_NAME=$(echo $TITLE | sed -e 's/[<>:"/\|?*]/_/g')
```

Okay, I've found a list of forbidden characters here: https://stackoverflow.com/a/31976060
That should do it 👍